src/Components/ProductsList/ProductsList.js

```diff
@@ -0,0 +1,51 @@
+import { React, useState, useEffect } from "react";
+import { useProduct } from "../../contexts/ProductProvider";
```

Comment:
try barrel export, the way you have done for components

Reply:
In next PR.